docs/modules/operation/pages/deep-dive/service-assurance/scheduled-outages.adoc

```diff
@@ -0,0 +1,84 @@
+
+= Scheduled Outages
+
+Scheduled outages let you suppress notifications and pause status polling, threshold checking, and data collection during a specified time period.
+
+== Some examples for why you would want to schedule an outage:
+** Updating components, scheduled reboots
+** Planned decommissioning of nodes
+** Outside influences like, power-, internet-, or other unplanned maintenance
+** Avoid excessive notifications to on-call staff
+** Scheduled infrastructure testing
+** Other events where thresholds could be reached or cause outages which could be considered normal
+
+To configure scheduled outages in {page-component-title}, click the *gear* symbol at the top-right of the page and, under Service Monitoring, select *Configure Scheduled Outages*.
+
+== Create a scheduled outage
```

Comment:
Suggest moving the contents of this section to the top of the "Creating a scheduled outage" section below (lines 18-52 -> 55).